feature-pack/wf-modules/src/main/resources/modules/org/infinispan/hibernate-search/directory-provider/for-hibernatesearch-5.10/module.xml

```diff
@@ -23,8 +23,8 @@ and obviously needs to avoid dependencies to different versions of Hibernate Sea
     </resources>
 
     <dependencies>
-        <module name="org.apache.lucene" slot="5.5" />
-        <module name="org.hibernate.search.engine" slot="@hibernate.search.module.slot@" />
+        <module name="org.apache.lucene"/>
```

Comment:
I am surprised this was removed and none of the tests failed...Does Wildfly 14 include lucene 5.5?

Reply:
Also, this module is to be linked against Wildfly internal hibernate search engine (as of WF 13, it's 5.10), and should not use the main lucene and hsearch slots, but specific versions hard-coded as search 5.10

Reply:
Yes: WildFly has made 5.10 the default (main slot) and included Lucene 5.5. There is also a 5.10 slot, but instead of being an alias, it contains a copy of the jars, thus causing LinkageErrors. That is why we have to use the main slot here.